src/main/java/net/dries007/tfc/world/region/ChooseBiomes.java

```diff
@@ -18,8 +18,8 @@ public enum ChooseBiomes implements RegionTask
     private static final int[] OCEANIC_MOUNTAIN_ALTITUDE_BIOMES = {VOLCANIC_MOUNTAINS, VOLCANIC_OCEANIC_MOUNTAINS, VOLCANIC_OCEANIC_MOUNTAINS, OCEANIC_MOUNTAINS, OCEANIC_MOUNTAINS, ROLLING_HILLS};
     private static final int[][] ALTITUDE_BIOMES = {
         {PLAINS, PLAINS, HILLS, HILLS, ROLLING_HILLS, LOW_CANYONS, LOWLANDS, LOWLANDS}, // Low
-        {PLAINS, HILLS, ROLLING_HILLS, HIGHLANDS, INVERTED_BADLANDS, BADLANDS, PLATEAU, CANYONS, LOW_CANYONS}, // Mid
-        {HIGHLANDS, HIGHLANDS, HIGHLANDS, ROLLING_HILLS, BADLANDS, PLATEAU, PLATEAU, OLD_MOUNTAINS, OLD_MOUNTAINS}, // High
+        {PLAINS, HILLS, ROLLING_HILLS, ROLLING_HILLS, ROLLING_HILLS, HIGHLANDS, BUTTES, MESAS, BADLANDS, PLATEAU_WIDE, CANYONS, CANYONS, LOW_CANYONS}, // Mid
+        {HIGHLANDS, HIGHLANDS, HIGHLANDS, HIGHLANDS, ROLLING_HILLS, ROLLING_HILLS, BADLANDS, BADLANDS, STAIR_STEP_CANYONS, PLATEAU, PLATEAU, PLATEAU, PLATEAU_WIDE, OLD_MOUNTAINS, OLD_MOUNTAINS, OLD_MOUNTAINS, OLD_MOUNTAINS}, // High
```

Comment:
I wonder when and if it is time to review the relative frequencies of each biome, but I don't have a problem with this

Reply:
With this PR, most of the biomes that I wanted to have are done. I did a bit of fiddling with the weighting to make sure it wasn't ridiculous, and checked distribution in the region generator, but haven't got an extensive sense of how common everything is yet. 

And since I said "most" of the biomes... my worldgen ambitions at this point are:
- Make volcanoes behave better
- Stratovolcano mountain variant (contingent on getting the volcanoes to behave better)
- New shore biomes/improving existing
- Peat bogs (with a few unique plants, and probably also bog iron deposits)